enterprise-suite/es-monitor-api/prometheus.yml

```diff
@@ -134,7 +130,8 @@ scrape_configs:
       - action: labelmap
         regex: __meta_kubernetes_service_label_(.+)
       - source_labels: [__meta_kubernetes_namespace]
-        action: replace
+        target_label: namespace
+      - source_labels: [__meta_kubernetes_namespace]
```

Comment:
It's okay that particular source_labels line appears twice?

Reply:
yep, I want to normalize everything on `namespace` but we have one hanger on in the UI that needs the `kubernetes_namespace` tag for now at least.  Dupe tags are ok since they don't change the number of metrics.